### ISSUE TITLE
Some properties does not work as intended

### ISSUE THREAD

**Issue by leon2968** [notfood/RimWorld-Infused#5]
Hi, 
I want to thank you first for your brilliant work on this mod. I enjoyed this as it gives items more uniqueness.

However, I notice some properties do not work as intended on melee weapons. The ones I found are below:
+ X Melee damage per second
X% Melee damage per second

While it shows the melee weapon has more dps with these properties, it doesn't actually improve the pawn's melee dps. For example, a Destructive Longsword shows more dps than a Longsword of Moonlight, but the pawn has the same dps while equipping those two weapons. I guess those properties don't really change the actual dps, they merely bump up the numbers,  Only the following properties do affect actual dps on the pawn:
X% Melee cooldown
X% Melee hit chance

**Comment by notfood** [comment]
Are you playing with Combat Extended? It will show but it'll do nothing, there is a branch for CE.

**Comment by leon2968** [comment]
No, I played it without CE. I changed the meleeweapon_averageDPS to meleeweapon_damagemultiplier and it works. You can see that the damagemultiplier actually boosts the damage done by smashing/cutting/piercing/jabbing/etc, while averageDPS does not actually change the damage done by those. Also, damagemultiplier actually affects the pawn's melee DPS. I hope this information helps =)
